poppy/utils.py

```diff
@@ -1465,7 +1468,7 @@ def specFromSpectralType(sptype, return_list=False, catalog=None):
         import os
         cdbs = os.getenv('PYSYN_CDBS')
         if cdbs is None:
-            raise EnvironmentError("Environment variable $PYSYN_CDBS must be defined for stsynphot")
+            raise EnvironmentError("Environment variable $PYSYN_CDBS must be defined for synphot")
```

Comment:
Does `PYSYN_CDBS` also need to be defined for synphot? Or should this line stay stsynphot? I thought it was stsynphot specific, but I may be wrong